### PR TITLE
Moved mods.json and reorganized some data (will require a fix on pss)

### PR DESCRIPTION
**Pull request by Su386yt**

### CODE REVIEW

File: data/mods.json [hunk 1]
```diff
@@ -1,261 +1,297 @@
...
       }
     },
     "spark": {
+      "name": "Spark",
       "download": "https://github.com/nea89o/spark",
       "versions": {
         "1.10.37+nea": "477a2ecf0b53e2d9d9d5b620a2d7177e81c4204055126e608d8e89d9647887be"
       }
     },
     "Dulkir Mod": {
```
Comment (j10a1n15) [inline data/mods.json:112]: why are some ids from uppercase to lowercase and some not

Comment (Su386yt) [reply data/mods.json:112]: I stopped halfway through and decided I'm gonna do this with the discord bot automatically

Comment (j10a1n15) [reply data/mods.json:112]: Ah I see

**Comment by Su386yt** [comment]
is it good

**Comment by j10a1n15** [comment]
Im sure you know what you are doing

**Pull request merged**